tests/testthat/_snaps/dummy.md

```diff
@@ -145,6 +145,15 @@
       Caused by error:
       ! `x` contains too many levels (123456), which would result in a data.frame too large to fit in memory.
 
+# throws an informative error for single level
+
+    Code
+      recipe(~., data = data.frame(x = "only-level")) %>% step_dummy(x) %>% prep()
+    Condition
+      Error in `step_dummy()`:
+      Caused by error in `bake()`:
```

Comment:
This feels inverted to me? i.e. `bake()` > `step_dummy()`

Reply:
True. ideally it would be switched, but i don't know how possible that is without causing a lot of issues

this is all handled by `recipes_error_context()` https://github.com/search?q=repo:tidymodels/recipes%20recipes_error_context&type=code